Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -23127,4 +23127,107 @@
         <Amount value="25"/>
         <Death value="Eviscerate"/>
     </CEffectDamage>
+    <CEffectSet id="AP_ArchonSiphon">
+        <EditorCategories value="Race:Protoss"/>
+        <EffectArray value="AP_ArchonSiphonSet"/>
+        <EffectArray value="AP_ArchonSiphonApply"/>
+    </CEffectSet>
+    <CEffectSet id="AP_ArchonSiphonSet">
+        <EditorCategories value="Race:Protoss"/>
+        <EffectArray value="AP_ArchonSiphonDamage"/>
+        <EffectArray value="AP_ArchonSiphonModify"/>
+    </CEffectSet>
+    <CEffectApplyBehavior id="AP_ArchonSiphonApply">
+        <EditorCategories value="Race:Protoss"/>
+        <WhichUnit Effect="AP_ArchonSiphonPersistent" Value="Caster"/>
+        <Behavior value="AP_ArchonSiphon"/>
+    </CEffectApplyBehavior>
+    <CEffectDamage id="AP_ArchonSiphonDamage">
+        <EditorCategories value="Race:Protoss"/>
+        <ResponseFlags index="Acquire" value="1"/>
+        <ResponseFlags index="Flee" value="1"/>
+        <Amount value="5"/>
+    </CEffectDamage>
+    <CEffectModifyUnit id="AP_ArchonSiphonModify">
+        <EditorCategories value="Race:Protoss"/>
+        <ImpactUnit Value="Caster"/>
+        <VitalArray index="Shields">
+            <Change value="10"/>
+        </VitalArray>
+    </CEffectModifyUnit>
+    <CEffectCreatePersistent id="AP_ArchonSiphonPersistent">
+        <EditorCategories value="Race:Protoss"/>
+        <WhichLocation Value="TargetUnit"/>
+        <Flags index="Channeled" value="1"/>
+        <InitialEffect value="AP_ArchonSiphon"/>
+        <PeriodCount value="15"/>
+        <PeriodicValidator value="AP_TargetNotDeadAndCasterNotDead"/>
+        <PeriodicEffectArray value="AP_ArchonSiphonSet"/>
+        <PeriodicPeriodArray value="0.125"/>
+    </CEffectCreatePersistent>
+    <CEffectDamage id="AP_ArchonSuicideDamage">
+        <EditorCategories value=""/>
+        <AttributeBonus index="Armored" value="50"/>
+        <AttributeBonus index="Massive" value="125"/>

```

Comment:
Doesn't work as intended: Thors will receive 162 damage (if multiple attribute bonuses happen, you get average the damage). Sky Furies have a workaround there with multiple damage effects so they stack properly
You need some better dmg handling there

What's the intended dmg to:
- Armored but not Massive?
- Massive but not Armored?
- Both Massive and Armored

Reply:
Changed to 75 + 75 to massive. The different bonuses to armored and massive were somewhat random. Easier to just change it, will probably be rebalanced in polishing stage anyway.